sdk/provisioning/Azure.Provisioning/src/Primitives/BicepValueReference.cs

```diff
@@ -36,6 +37,5 @@ internal BicepExpression GetReference()
         return target;
     }
 
-    public override string ToString() =>
-        $"<{GetReference()} from {Construct.GetType().Name}.{PropertyName}>";
+    public override string ToString() => GetReference().ToString();
```

Comment:
Is it ok that this throws when the construct is not named?

Reply:
I was debating this.  I think I'm the only person who will ever look at this type, but I'll patch this to be friendlier.